src/Kernel/ConstantBlockClosure.class.st

```diff
@@ -21,6 +23,21 @@ Class {
 	#category : #'Kernel-Methods'
 }
 
+{ #category : #'instance creation' }
+ConstantBlockClosure class >> numArgs: numArgs literal: aLiteral [
+	| classToUse |
+	classToUse := numArgs caseOf:
+	{
+		[0]->[ self ].
```

Comment:
I think it would be more clear to have explicit ConstantBlockClosure0Arg class and keep ConstantBlockClosure as abstract

Reply:
I add it to the TODO, I was thinking the same. Maybe adding the 4 arg version is good, too (to have all the ones coverd that we can call without using #valueWithArguments:)